app/templates/install/index.stimulus.js

```diff
@@ -14,7 +14,7 @@ context.keys().forEach((path) => {
   //   nav/user_info/index.js -> nav--user-info
   const identifier = path.replace(/^\\.\\//, '')
     .replace(/\\/index\\.js$/, '')
-    .replace(/\\//, '--');
```

Comment:
I'm not sure why the regexp was `/\\//` instead of `/\//` here?

Reply:
As far as I remember, we need to escape `\`, since this is a template file we pass through ERB.